Block/BlockContextManager.php

```diff
@@ -294,7 +295,12 @@ private function resolve(BlockInterface $block, $settings)
         }
 
         if ($this->reflectionCache[$serviceClass]['isOldOverwritten'] && !$this->reflectionCache[$serviceClass]['isNewOverwritten']) {
-            @trigger_error('The Sonata\BlockBundle\Block\BlockServiceInterface::setDefaultSettings() method is deprecated since version 2.3 and will be removed in 4.0. Use configureSettings() instead. This method will be added to the BlockServiceInterface with SonataBlockBundle 4.0.', E_USER_DEPRECATED);
+            @trigger_error(
+                'The Sonata\BlockBundle\Block\BlockServiceInterface::setDefaultSettings() method is deprecated'
+                .' since version 2.3 and will be removed in 4.0. Use configureSettings() instead.'
```

Comment:
so what would be the result in this case?
It looks semantically to me @greg0ire